mac/deploy_mac.sh

```diff
@@ -181,6 +172,27 @@ build_installer_image() {
         "${deploy_path}/"
 }
 
+build_storesign_pkg() {
+    # Build installer pkg file - for submission to App Store
+    # Note: We do not upload the server to the app store for now. This could be changed easily by uncommenting the respective lines below
+    local client_target_name="${1}"
+    # local server_target_name="${2}"
+    echo "Cert signing for App Store started..."
+
+    # Copy binaries to separate temporary deploy directory leave the (adhoc) signed app untouched
+    cp -a "${raw_path}/${client_target_name}.app" "${macapp_deploy_path}/${client_target_name}.app"
+    # cp -a "${deploy_path}/${server_target_name}.app" "${macapp_deploy_path}/${server_target_name}.app"
```

Comment:
Is it correct that client uses `raw_path` but server uses `deploy_path`? Or should they both be the same?

I can see from line 110 above that one contains a copy of the other, but I'm not clear whether at this point they are still the same.

Reply:
No. That's an oversight from me.

Reply:
Done.